.appveyor.yml

```
@@ -9,21 +9,23 @@ image:
 - Visual Studio 2022
 - Visual Studio 2019
 clone_depth: 50
-init:
+install:
+  - cinst make
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)